docs/src/js/base/listbox.js

```diff
@@ -37,11 +41,11 @@ function onKeyDown(e) {
     if (e.keyCode === 13 || e.keyCode === 32) { // enter key or spacebar key
         const toElIndex = this._activeDescendant.index;
         const toEl = this.items[toElIndex];
-        const isTolElChecked = toEl.getAttribute('aria-checked') === 'true';
+        const isTolElSelected = toEl.getAttribute('aria-selected') === 'true';
```

Comment:
Why create a variable if it's not being used? I know this was old code but since it is being touched might as well do:
if (toEl.getAttribute('aria-selected') !== 'true') {
   //etc
}

Reply:
Also I notice that this is basically doing the same as the onclick. Should we combine them in a function and just pass in arguments to perform the given actions.

Reply:
Yep, you're right, there's a bunch of duplicate logic there. I plan on going through and cleaning up listbox and listbox-button.js thoroughly some other time, but probably not in this PR.